kamon-metrics/src/main/scala/com/ovoenergy/effect/Metrics.scala

```diff
@@ -24,9 +25,15 @@ object Metrics {
   /**
    * An instance of metrics for a Sync[F] which wraps the underlying calls
    */
-  def syncKamonMetrics[F[_]: Sync: FlatMap]: Metrics[F] =
-    metric =>
+  implicit def SyncMetrics[F[_]: Sync: FlatMap]: Metrics[F] = new Metrics[F] {
```

Comment:
also Sync implies FlatMap so you can remove it

Reply:
And I'd lowercase the S

Reply:
yep, you're absolutely right about FlatMap (also I didn't see the `s` AT ALL)